src/Command/Import_Command.php

```
@@ -0,0 +1,717 @@
+	/**
+	 * Sets the params based on the command line arguments.
+	 *
+	 * @since 1.0.0
+	 * @version 1.0.0
+	 *
+	 * @param array<int, string> $args       The command arguments.
+	 * @param array<int, string> $assoc_args The command associative arguments.
+	 *
+	 * @return void
+	 */
+	private function set_params( array $args, array $assoc_args ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)